src/services/ProjectsService.ts

```diff
@@ -84,12 +85,32 @@ const searchProjects = async (organizationId: number, query?: string): Promise<P
   return response ? (response as Project[]) : null;
 };
 
+/**
+ * Create a project
+ */
+const createProject = async (project: CreateProjectRequest): Promise<Response> => {
+  return await httpProjects.post({
+    entity: project,
+  });
+};
+
+type AssignProjectRequestPayload = components['schemas']['AssignProjectRequestPayload'];
+
+const assignProjectToEntities = async (projectId: number, entities: AssignProjectRequestPayload) =>
+  httpProjects.post({
```

Comment:
await

Reply:
I think the solve is to actually remove `async`. Awaiting here doesn't do anything unless I am expecting to affect the control flow within this function, but we are just pre-baking the HTTP call.